contracts/price/ZNSCurvePricer.sol

```diff
@@ -175,22 +179,13 @@ contract ZNSCurvePricer is AAccessControlled, ARegistryWired, UUPSUpgradeable, I
         emit MaxPriceSet(domainHash, maxPrice);
     }
 
-    /**
-     * @notice Sets the minimum price for domains. Validates the config with the new price.
-     * Fires `MinPriceSet` event.
-     * Only domain owner/operator can call this function.
-     * @param domainHash The domain hash to set the `minPrice` for
-     * @param minPrice The minimum price to set in $ZERO
-     */
-    function setMinPrice(
+    function setCurveMultiplier(
         bytes32 domainHash,
-        uint256 minPrice
+        uint256 curveMultiplier
     ) external override onlyOwnerOrOperator(domainHash) {
-        priceConfigs[domainHash].minPrice = minPrice;
+        priceConfigs[domainHash].curveMultiplier = curveMultiplier;
```

Comment:
do you have any limitations on what this value could be?
Are there any values that would break the formula if set as `curveMultiplier`?
If you have limits for this value, it would be good to add checks here that will revert if incorrect value is set.

Reply:
- `curveMultiplier` = 1.000 - default. Makes a canonical hyperbola (regular).
- It can be "0", which makes all domain prices max.
- If it is less than 1.000, then it pulls the bend towards the straight line.
- If it is bigger than 1.000, then it makes bigger slope on the chart.

I'm adding these comments to contract to be clear